apps/openmw/mwgui/tradewindow.hpp

```diff
@@ -65,8 +65,10 @@ namespace MWGui
 
             int mItemToSell;
 
-            int mCurrentBalance;
-            int mCurrentMerchantOffer;
+            bool mIsPlayerSelling;
+            bool mIsMerchantBuying;
```

Comment:
The thing is that with the current code you can't decrease the offer to 0, and if it happens to be an even trade of items then you can't increase the offer. It's a minor difference but the original game does let you haggle to 0.